doc/contents.md

```diff
@@ -0,0 +1,31 @@
+DLite documentation
+===================
+
+This is a temporary table of contents, used during the development phase.
+When completed, the table of contents will be written in pydoc/index.rst
+and built with sphinx.
+
+Table of contents
+-----------------
+
+1. Introduction
+    * What is DLite and what is it used for?
+    * [Features](features.md)
+2. Installation instructions
+3. [Build instructions](build_with_vs.md)
+4. Tutorial
+5. Basic topics
+    * [Concepts](concepts.md)
+    * Data types
+    * [Mappings](mappings.md)
+    * [Collections](collections.md)
+    * Bindings
+6. [Tips and tricks](tips_and_tricks.md)
```

Comment:
The tips_and_tricks.md is currently for developers. 
I will suggest to add a "Developers guide" after "Advanced topics" under which we can have the documentation guideline and tips and tricks.

Reply:
Implemented as suggested.